apps/ledger-live-desktop/src/renderer/families/celo/RevokeFlowModal/steps/StepVote.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import invariant from "invariant";
+import React, { useCallback, useMemo } from "react";
+import { Trans } from "react-i18next";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Box from "~/renderer/components/Box";
+import Button from "~/renderer/components/Button";
+import ErrorBanner from "~/renderer/components/ErrorBanner";
+import RevokeVoteRow from "../components/RevokeVoteRow";
+import type { StepProps } from "../types";
+import { useCeloPreloadData } from "@ledgerhq/live-common/lib/families/celo/react";
+import { getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import { revokableVotes } from "@ledgerhq/live-common/lib/families/celo/logic";
+import Alert from "~/renderer/components/Alert";
+import { urls } from "~/config/urls";
+
+export default function StepVote({
+  account,
+  parentAccount,
+  onChangeTransaction,
+  transaction,
+  error,
+}: StepProps) {
+  invariant(
+    account && account.celoResources && transaction,
+    "celo account, resources and transaction required",
+  );
+
+  const bridge = getAccountBridge(account, parentAccount);
+
+  const onChange = useCallback(
+    (recipient: string, index: number) => {
+      onChangeTransaction(
+        bridge.updateTransaction(transaction, {
+          recipient,
+          index,
+        }),
+      );
+    },
+    [bridge, transaction, onChangeTransaction],
+  );
+
+  const votes = revokableVotes(account);
+
+  if (!transaction.recipient && votes[0]) onChange(votes[0].validatorGroup, votes[0].index);
+
+  const { validatorGroups } = useCeloPreloadData();
+
+  const mappedVotes = useMemo(
+    () =>
+      votes?.map(vote => ({
+        vote,
+        validatorGroup: validatorGroups.find(v => v.address === vote.validatorGroup),
+      })) || [],
+    [votes, validatorGroups],
+  );
+
+  const unit = getAccountUnit(account);
+
+  return (
+    <Box flow={1}>
+      <TrackPage category="Withdraw Flow" name="Step 1" />
```

Comment:
Similar comment as [here](https://github.com/pawelnguyen/ledger-live/pull/1/files#r889211323) - should the category be "Celo Revoke Vote"?